test_monitor_window.py

```
@@ -0,0 +1,150 @@
+            test_status["reels"]["reel_0001"]["completed"] = True
+            test_status["monitoring"] = False
+
+            with open(status_file, 'w') as f:
+                json.dump(test_status, f, indent=2)
+
+            print("   Updated status to show completion - window should close automatically")
+
+            # Clean up
+            try:
+                process.terminate()
+            except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)